include/votca/xtp/aotransform.h

```diff
@@ -23,6 +23,10 @@
 
 #include <votca/xtp/aoshell.h>
 #include <votca/xtp/eigen.h>
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wshadow"
```

Comment:
@JensWehner that is for you to fix ;-)

Reply:
okay? What happened there?

Reply:
Just remove the block and compile with `-Wshadow` and have look at he warnings.

Reply:
It might be fixed with `enum class`.